### PR TITLE
Add details on how to get set up for development

### PR DESCRIPTION
**Pull request by garry-jeromson**
This should (hopefully) take any brainwork out of being able to run most of the Makefile targets.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,17 +69,39 @@ def mock_server(request):
...
+pip install pyenv tox tox-pyenv
+pyenv install 3.5.9
+pyenv install 3.8.2
+pyenv local 3.5.9 3.8.2
+```
+
+In order to run `make test`, you'll also need to have Mountebank installed locally:
+
+```sh
+npm install mountebank
```
Comment (brunns) [inline README.md:92]: Can we pin the version, as on line 34?

File: src/mbtest/imposters/predicates.py [hunk 2]
```diff
@@ -58,6 +58,7 @@ class Method(Enum):
         HEAD = "HEAD"
         POST = "POST"
         PUT = "PUT"
+        PATCH = "PATCH"
```
Comment (brunns) [inline src/mbtest/imposters/predicates.py:61]: This might be better as a separate PR?

File: README.md (hunk 1 above)
Comment (garry-jeromson) [reply README.md]: Done.

File: src/mbtest/imposters/predicates.py (hunk 2 above)
Comment (garry-jeromson) [reply src/mbtest/imposters/predicates.py]: Aye - done.
